docs/step_by_step_guide.md

```diff
@@ -0,0 +1,647 @@
+```
+get_dates()
+{
+    for f in atm.log.*.gz; do
+        echo $f
+        zgrep "DATE=" $f | head -n 1
+        zgrep "DATE=" $f | tail -n 1
+	echo ""
+    done
+}
+```
+(If `zgrep` is unavailable, use `less <log>` to look at a gzipped log file. Scroll down a decent amount to `DATE=` to find the start date. Use `SHIFT+g` to go to the end of the file. Scroll up to `DATE=` to find the end date.)
+
+In the “Years” column specify `<start> - <end>`, with each in `year-month-day` format.
+
+To find the number of nodes, first look at the Processor # / Simulation Time chart on PACE. The x-axis lists the highest MPI rank used, with base-0 numbering of ranks. (PE layouts often don’t fit exactly `N` nodes but instead fill `N-1` nodes and have some number of ranks left over on the final node, leaving some cores on that node unused). Then, find `MPI tasks/node` in the “Experiment Details” section. The number of nodes can then be calculated as `ceil((highest MPI rank + 1)/(MPI tasks/node))`.
+
+The SYPD (simulated years per day) is listed in PACE’s “Experiment Details” section as `Model Throughput`.
+
+In the “Notes” section of the table, mention if a job failed or if you changed anything before re-running a job.
+
+## Global Time Series
+> [!NOTE]
+> The plots will be available online at the URL corresponding to `<www>/global_time_series/` (where `www` is specified in the `zppy` cfg). See the [E3SM Diags quick guide](https://e3sm-project.github.io/e3sm_diags/_build/html/master/quickguides/quick-guide-general.html) to find the URLs for the web portals on each E3SM machine (listed as `<web_address>`).
+
+You can download the images and then upload them to your Confluence page.
+
+## E3SM Diags
+> [!NOTE]
+> The plots will be available online at the URL corresponding to `<www>/e3sm_diags/` (where `www` is specified in the `zppy` cfg). See the [E3SM Diags quick guide](https://e3sm-project.github.io/e3sm_diags/_build/html/master/quickguides/quick-guide-general.html) to find the URLs for the web portals on each E3SM machine (listed as `<web_address>`).
+
+Replace the baseline diagnostics in the template's table with relevant ones (e.g., diags on v3 `piControl` and another relevant `v3` run). Add your own diagnostics links in the last columns, labeling them as `<start_year>-<end_year>`.
+
+## MPAS Analysis
+
+> [!NOTE]
+> The plots will be available online at the URL corresponding to `<www>/mpas_analysis/` (where `www` is specified in the `zppy` cfg). See the [E3SM Diags quick guide](https://e3sm-project.github.io/e3sm_diags/_build/html/master/quickguides/quick-guide-general.html) to find the URLs for the web portals on each E3SM machine (listed as `<web_address>`).
+
+Make a bulleted list of links, e.g., for `<url_path>/ts_0001-0050_climo_0021-0050/`, create a bullet `“1-50 (time series), 21-50 (climatology)”.
+
+## ILAMB
+<!-- TODO: Add this section? -->
```

Comment:
The original guide was written with Water Cycle in mind. Is there a preferred way to include ILAMB links in simulation documentation?